pkg/grpc/server/transactions_api.go

```diff
@@ -222,7 +223,12 @@ func (s *Server) Broadcast(ctx context.Context, tx *pb.SignedTransaction) (out *
 	if err != nil {
 		return nil, apiError(err)
 	}
-	t, err = t.Validate(s.scheme)
+	lightNodeActivated, err := s.state.IsActivated(int16(settings.LightNode))
```

Comment:
`IsActivated` used `features.newestIsActivated`. Is it correct to use `Newest` methods here?

Reply:
I think it doesn't matter. It's a user API, transaction doesn't get to blockchain directly from here, it will be validated once more (at least) in UTX during block generation.